tsconfig.json

```diff
@@ -3,18 +3,19 @@
     "compilerOptions": {
         "downlevelIteration": true,
         "sourceMap": false,
-        "target": "es5",
-        "module": "commonjs",
-        "moduleResolution": "node",
+        "target": "ES2020",
+        "module": "ESNext",
+        "moduleResolution": "Node",
         "importHelpers": true,
         "removeComments": true,
-        "noImplicitAny": true,
```

Comment:
This `noImplicitAny` option is now less strict. Is this a must for running with WDIO v8? If it isn't, I would re-enable this option to force us to have a more typed code in this library.